### ISSUE TITLE
Search input is *not* rounded-md on iOS Chrome (Home Screens - Constrained Multi Column)

### ISSUE THREAD

**Issue by kendalled** [tailwindlabs/tailwindui-issues#305]
**What component (if applicable)**
- URL for category: https://tailwindui.com/components/application-ui/page-examples/home-screens
- Component name: Constrained Multi Column

**Describe the bug**
The search input at the top of the page is not rounded-md like on all other breakpoints. I could not get this behavior to reproduce in the browser devtools, I had to access the site on my iPhone XS, Chrome (Latest Version). The search input seems to be rounded-lg, as it is pill shaped (seen below). However, once you focus the input or start typing, rounded-md seems to get applied again. 

I assume this is something to do with tailwind-custom-forms, as adding appearance-none to the search input fixed the behavior. I assume this just does a style reset and then the rounded-md applies all of the time, as intended.

Screenshots are below since this is a weird issue, and cannot be reproduced on MacOS Chrome by making the device an iPhone. I assume it's something to do with user-agent stylesheets on mobile as well. 

**To Reproduce**
1. Go to 'Constrained Multi Column' & Copy code
2. Deploy to any hosting / server (netlify, firebase, etc)
3. Visit page on iPhone, Chrome Browser
4. See pill-shaped input (error part 1 - should be rounded-md)
5. Focus the search and see it change to rounded-md (error part 2 - should not change border radius on focus / input)

**Expected behavior**
A search input that is rounded-md across all devices and browsers. (Sorry if harsh, this section always makes me feel bad)

**Screenshots**
![IMG_1169](https://user-images.githubusercontent.com/22782727/91891520-2d0b8780-ec5f-11ea-8a8f-be74ed31ba93.PNG)
![IMG_1170](https://user-images.githubusercontent.com/22782727/91891556-3bf23a00-ec5f-11ea-8144-01acea61d418.PNG)

Screenshot one: Unedited (besides text) source code uploaded to firebase hosting (ssr). See pill-shaped input. 
Screenshot two: Focused on input, rounded-md gets applied. Also stays rounded-md if any input is entered.

**Browser/Device (if applicable)**
 - OS: iOS (Latest)
 - Browser: Chrome
 - Version: Latest
- Device: iPhone XS

**Additional context**
Sorry if it's a bit wordy, but this issue is very specific & a little hard to describe. However, the solution does appear to be as simple as adding 'appearance-none' to the search input - so it becomes:

              `<input id="search" class="appearance-none block w-full pl-10 pr-3 py-2 border border-transparent rounded-md leading-5 bg-indigo-400 bg-opacity-25 text-indigo-300 placeholder-indigo-300 focus:outline-none focus:bg-white focus:placeholder-gray-400 focus:text-gray-900 sm:text-sm transition duration-150 ease-in-out" placeholder="Search projects" type="search">`

Hope this helps - love the work on TailwindUI so far. I am unaware if this issue exists in other components, as I have not checked. Will update if I notice any more.

Thanks,
Kendall Jackson
Orlando, FL

**Comment by davidluhr** [comment]
@kendalled Thank you for reporting this. Very strange and specific bug indeed.

The recent update to Tailwind UI comes with a new forms plugin from Tailwind CSS 2.0 that should make styling form inputs easier and more consistent. Can you let us know if this issue still persists on the latest version of Tailwind UI?

**Comment by davidluhr** [comment]
@kendalled I'm combing back through open issues to see if I can help in any way. I'm closing this issue for now, as I haven't seen an update, but please don't hesitate to respond if this issue still exists and we can re-open this.

**Comment by kendalled** [comment]
Sorry for the late response David, but yes, the Forms plugin fixed the issue. Thanks for looking into this.
